static/schemas/2.0/node.2.0.json

```diff
@@ -25,6 +25,10 @@
                         "type": "object"
                     }
                 },
+                "snmpSettings": {
```

Comment:
Isn't the snmpSettings has been moved into `ibms` or `obms`? @brianparry @srinia6